tests/tas-mtls/src/test/resources/default.properties

```diff
@@ -0,0 +1,2 @@
+# dataprep related
+alfresco.port=8082
```

Comment:
A question, I remember all the woes faced to fix that port issue, buut, couldn't we change the port in docker compose instead to fit to the one provided by default?

Reply:
I meant the exposed port in docker compose, in the "ports" section where we decide what it should be, not the property value. Though granted they would need to change too.